openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -340,7 +340,7 @@ tools/pack200/Pack200Test.java https://github.com/adoptium/aqa-tests/issues/2657
 tools/launcher/RunpathTest.java https://bugs.openjdk.java.net/browse/JDK-8286118 linux-arm
 tools/launcher/VersionCheck.java https://github.com/adoptium/aqa-tests/issues/4243 linux-x64
 sun/jvmstat/monitor/MonitoredVm/MonitorVmStartTerminate.java https://github.com/adoptium/infrastructure/issues/2565 solaris-x64
-sun/jvmstat/monitor/MonitoredVm/CR6672135.java https://github.com/adoptium/aqa-tests/issues/3691 windows-x64
+sun/jvmstat/monitor/MonitoredVm/CR6672135.java https://github.com/adoptium/aqa-tests/issues/3691,https://bugs.openjdk.org/browse/JDK-8062938 generic-all
```

Comment:
```suggestion
sun/jvmstat/monitor/MonitoredVm/CR6672135.java https://bugs.openjdk.org/browse/JDK-8062938 generic-all
```

Reply:
I reserved 'https://github.com/adoptium/aqa-tests/issues/3691' becasue I think if [JDK-8062938](https://bugs.openjdk.org/browse/JDK-8062938) backported to jdk8u-dev in future, we can recover this Problemlist entry from `generic-all` back to `windows-x64`.
But I think remove the original link is acceptable also.